internal/e2e/integration_test.go

```diff
@@ -0,0 +1,292 @@
+// Copyright 2024 SolarWinds Worldwide, LLC. All rights reserved.
```

Comment:
`internal/e2e/integration_test.go`: Is it `integration` or `e2e`? I think it can be both.

Reply:
yop, from the collector's perspective it can be e2e, yet the surrounding `integration` term usage around is predominant indeed. I think with naming we might even go after what it acually tests - sort of `signals_processing_test.go` or similar as that's what we are interested in in the end. 

Still, we should make the final call and potentially tidy up in the build tags, job names etc. so there is clear line drawn of what this level is (and is supposed to be contained within the package).